### PR TITLE
Please have a look

### PR DESCRIPTION
**Pull request by thapakrishna0111**
Please check this pull request

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1: abc
+
+config2: 123
```
Comment (krishnathapa90) [inline util/config.txt:3]: Please check this config before approving.

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
```
Comment (krishnathapa90) [inline util/feature.txt:3]: is this implementation is correct or not?

File: util/utility.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+This is a utility
+
+there are several different features
+1
+2
+3
+
+and multiple configs
```
Comment (krishnathapa90) [inline util/utility.txt:8]: this line has some issues please check once.

**Review by krishnathapa90 — CHANGES_REQUESTED** [review]
Please merge once checked

**Review by krishnathapa90 — COMMENTED** [review]
All good

**Comment by thapakrishna0111** [comment]
This merge pull request is complete

**Pull request merged**